example/src/App.tsx

```diff
@@ -87,7 +87,7 @@ export const App: React.FC = () => {
           bounciness: 100,
           speed: 1,
           toValue: Dimensions.get("screen").height * 0.25,
-          useNativeDriver: Platform.OS !== "android"
+          useNativeDriver: false
```

Comment:
I wonder, how will this impact performance on iOS device, and also how we can address any issues if the impact is meaningful

Reply:
Yeah, it seems things have changed with animations in new React Native versions so we need to revisit the animation workflow. This is just the example code but I think it's a good idea to leave the comment until this is solved 🙂